### ISSUE TITLE
empty contact links in person profiles: 2

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#294]
When the "phone" field in an isaw profile is empty, the template produces in the rendered HTML:

```html
<a href="mailto:"></a>
```

Better behavior would be to emit nothing if there is no phone number specified.

**Comment by alecpm** [comment]
All empty fields should be removed from the form now, along with the containing fields. Additionally, there was some questionable code used to render to "Blurb" which I've cleaned up.

**Comment by paregorios** [comment]
This looks good. Please deploy to production.

**Comment by paregorios** [comment]
this is working on production
